plugins/vcr.rst

```diff
@@ -0,0 +1,96 @@
+VCR Plugin
+==========
+
+The VCR plugins allow you to record & replay HTTP responses.
```

Comment:
should we add "This is mainly useful for testing purposes."? its obvious to you and me, but maybe not to everyone.